code/modules/events/aurora_caelus.dm

```diff
@@ -40,10 +42,10 @@
 			message_admins("Aurora Caelus event caused an oven to ignite at [ADMIN_VERBOSEJMP(ruined_roast)].")
 			log_game("Aurora Caelus event caused an oven to ignite at [loc_name(ruined_roast)].")
 			announce_to_ghosts(roast_ruiner)
-		for(var/mob/living/carbon/human/seymour as anything in GLOB.human_list)
-			if(seymour.mind && istype(seymour.mind.assigned_role, /datum/job/cook))
-				seymour.say("My roast is ruined!!!", forced = "ruined roast")
-				seymour.emote("scream")
+	for(var/mob/living/carbon/human/seymour as anything in GLOB.human_list)
+		if(seymour.mind && istype(seymour.mind.assigned_role, /datum/job/cook))
```

Comment:
Shouldn't this only happen if the chef can see kitchen oven, is alive and conscious? It makes zero sense for a chef to blurt this out while MIA or KIA, d'uh.

Reply:
yeah sure lemme add a check for that too